app/controllers/media_controller.rb

```diff
@@ -1,5 +1,4 @@
 class MediaController < BaseMediaController
-  skip_before_filter :require_signin_permission!
```

Comment:
Can the corresponding specs in `media_controller_spec.rb:265` be removed here too?

Reply:
Although superficially, [the example you mention](https://github.com/alphagov/asset-manager/blob/0c43e01220f272f2887c93d3880c989bb007c189/spec/controllers/media_controller_spec.rb#L265-L269) *looks* as if it is related to the `skip_before_filter`, it is in fact testing [the behaviour provided by `MediaController#authenticate_if_private](https://github.com/alphagov/asset-manager/blob/0c43e01220f272f2887c93d3880c989bb007c189/app/controllers/media_controller.rb#L58-L60) which is added by a `before_filter` in `MediaController`.

As far as I could see, although there are examples which rely on the skipping of the before filter, there are none that directly test for its existence. Thus I don't think there are any examples which can be removed.